src/binary/create-logger.ts

```diff
@@ -1,3 +1,7 @@
+import jsonStringify from 'fast-safe-stringify';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { MESSAGE } from 'triple-beam';
```

Comment:
I pull `fast-safe-stringify` and `triple-beam` to create the custom format (down below), however, we don't have those dependencies declared in our project. They're dependencies of `winston`. This is why I have to make `@ts-ignore` comment because ts compiler cannot find type declarations for `triple-bream`.


I think a good practice would be to not import from not-direct dependencies, however, maybe in this case we can make an exception. I'm not sure. Should we install them? WYT @willclarktech

Reply:
If we import them directly, let's add them to dependencies in `package.json`

Reply:
74ec8a4